### ISSUE TITLE
Origin?

### ISSUE THREAD

**Issue by etnt** [tonyrog/dhcp#2]
https://github.com/tonyrog/dhcp/blob/6153a8a3f4d6010c6b6eb1bc91552c12d15c0d80/AUTHORS#L1

Ha...just happen to stumble on this code. The origin to this code must be an implementation I and Magnus.F wrote at Nortel in early 2006 in a 2 weeks hack while visiting the San Jose office. Ruslan was our contact person at Nortel and seem to have made it public (good of him :-)

**Comment by tonyrog** [comment]
Ha ha!  I found this in the example dhcp.conf file ( {15,"us.nortel.com"} ) so you are correct. Do you want your name in the AUTHORS file? Please add it! If I remember correctly I needed it for an access point project at one point and found the code floating somewhere outside of github.

**Comment by etnt** [comment]
No, it doesn't matter. I was just surprised to find the code here ;-)

Funny story: Some week after we had left San Jose we heard from Ruslan that their test department had complained to him that our DHCP server only could handle 70% of the traffic they were sending to it. When Ruslan asked how much traffic they did send, he got the anser "everything we've got". Well, they decided to test the standard Linux DHCP server to get a baseline, but noted then that it only was capable of handling 30%...
